File: Beecrowd/1036 Fórmula de Bhaskara/main.cpp
#include <iostream>
#include <math.h> 
using namespace std;
 
int main() {
 
	double A, B, C, R1, R2, Delta;
	
	cin >> A;
	cin >> B;
	cin >> C;
		
	Delta = pow(B, 2) - 4 * A * C;
	R1 = (-B + sqrt(Delta)) / (2 * A);
	R2 = (-B - sqrt(Delta)) / (2 * A);
	
	if(Delta < 0 || A == 0){
		cout << "Impossivel calcular" << endl;
	}
	else{
		cout << fixed;
		cout.precision(5);
		cout << "R1 = " << R1 << endl;
		cout << "R2 = " << R2 << endl;
	}
 
    return 0;
}

File: Beecrowd/1036 Fórmula de Bhaskara/main.py
import math

entrada = input().split()
a = float(entrada[0])
b = float(entrada[1])
c = float(entrada[2])

delta = b * b - 4 * a * c

if delta < 0 or a == 0:
    print("Impossivel calcular")
else:
    x1 = (-b + math.sqrt(delta)) / (2 * a)
    x2 = (-b - math.sqrt(delta)) / (2 * a)
    print("R1 = %0.5f" %x1)
    print("R2 = %0.5f" %x2)